http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -387,6 +381,33 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       _ <- logger.info(ctx.loggingCtx)(
         s"Finished app creation for app ${app.appName.value} in cluster ${gkeClusterId.toString}"
       )
+
+      // TODO: this logic will be changed in the next PR
```

Comment:
How will this change? 

Right now it looks like:
- if the disk already has a PvcId saved (e.g. it was previously attached to a Galaxy) --> do nothing
- if the disk does NOT have a PvcId saved, fetch it from k8s and persist it

Reply:
it's mostly the criteria for determining whether we need to persist PVCIds could change. Since there's also `createDisk` flag, that could be used as well....I'm trying to not duplicating this info (also not duplicating `chart`, `release` in multiple places) somehow..I don't know exactly what it'll look like yet, hence just left a comment